osu.Game/Overlays/Music/PlaylistItem.cs

```diff
@@ -114,24 +117,58 @@ private void recreateText()
             });
         }
 
-        protected override bool OnHover(Framework.Input.InputState state)
+        protected override bool OnHover(InputState state)
         {
             handle.FadeIn(fade_duration);
 
             return base.OnHover(state);
         }
 
-        protected override void OnHoverLost(Framework.Input.InputState state)
+        protected override void OnHoverLost(InputState state)
         {
             handle.FadeOut(fade_duration);
         }
 
-        protected override bool OnClick(Framework.Input.InputState state)
+        protected override bool OnClick(InputState state)
         {
             OnSelect?.Invoke(BeatmapSetInfo);
             return true;
         }
 
+        protected override bool OnDragStart(InputState state)
+        {
+            return true;

```

Comment:
The items should only be draggable if the little cheeseburger thing to the left of the items is hovered.